compatibility/src/sway.rs

```diff
@@ -0,0 +1,85 @@
+// Credits to Sway in the Rust Programming Language
```

Comment:
perhaps include a link to the project. Also mention the license of sway

Reply:
I don't have a link to a project. The license is "I asked for permission and they said yes", is this something we can work with?

Reply:
Oh, was this not the [sway](https://swaywm.org/) project?

Reply:
If it was a random user, yes this is fine.